### ISSUE TITLE
Add an explicit license.

### ISSUE THREAD

**Issue by Skenvy** [dependabot/smoke-tests#17]
At the moment, the closest to this is the README saying
> You're welcome to use this repo to test Dependabot functionality.
But it would be nice if the licensing was less vague and ephemeral.

I ask because I'd like to duplicate the files here in a set of tests to run github's linguist on. Linguist can't run on submodules, and I'd like for the unit tests to rely on an external repo, or multiple versions of it given the pinned versions that the smoke tests here guarentee to be runnable with dependabot.

**Comment by Skenvy** [comment]
I hope this request for smoothing over the meta-context of the repository can bypass the
> However, we aren't accepting contributions at this time.

**Comment by jeffwidman** [comment]
Yeah, that's a reasonable request. I don't have full context here, but I recall when the repo was created that they weren't ready to choose a license and the guy most involved is currently out on paternity leave until end of January. Since this isn't urgent, I'll wait til he's back and then circle back with him.

**Comment by jeffwidman** [comment]
Yeah, that's totally fine for now. 

I don't know, but I suspect the internal question is just whether we need the prosperity license here too, or if we can keep it simple with MIT license. But in either case, that wouldn't prohibit what you're doing.

If for some reason we have to move to a license that doesn't allow copying, we can always just reach out to you down the road. But in the meantime please don't let this block what you're working on as long as it's work on a free OSS project.

**Comment by yeikel** [comment]
Now that `dependabot-core` [uses MIT](https://github.com/dependabot/dependabot-core/pull/9717), would it make sense to add MIT here as well and close this issue? 

I think that we might need to add an explicit `CONTRIBUTING.md` as well

**Comment by pombredanne** [comment]
Ping, this would be timely. No license at all on a GitHub repo is not a great practice.

**Comment by pombredanne** [comment]
@Jeffrey-Luszcz ping: is this something you may be able to help resolve?
